entrypoint.sh

```diff
@@ -4,12 +4,17 @@ if [ $# == 0 ]; then
     echo "Usage: url [grade] [followRedirects]"
     echo "* url: URL to analyse."
     echo "* grade: The desired security grade of your HTTP response headers. Possible grades: A+, A, B, C, D, E, F. Defaults to B."
-    echo "* followRedirects: Follow redirects. Defaults to false, set to true to enable."
+    echo "* followRedirects: Follow redirects. Defaults to on, set to off or false to disable."
     exit 1
 fi
 
+
 GRADE=${2:-'B'}
-FOLLOW_REDIRECTS=${3:-''}
+FOLLOW_REDIRECTS=${3-'1'}
+FOLLOW_REDIRECTS=${FOLLOW_REDIRECTS/true/1}
+FOLLOW_REDIRECTS=${FOLLOW_REDIRECTS/on/1}
+FOLLOW_REDIRECTS=${FOLLOW_REDIRECTS/false/0}
```

Comment:
I also made the value `'false'` mean `'off'` too because I think if I were configuring the action then I would expect to set the value to `false` in the yml.